test/e2e/policyrecommendation_test.go

```diff
@@ -72,35 +68,99 @@ func TestPolicyRecommendation(t *testing.T) {
 		testPolicyRecommendationDelete(t, data)
 	})
 
+	t.Run("testPolicyRecommendationFailed", func(t *testing.T) {
+		testPolicyRecommendationFailed(t, data)
+	})
+
 	podAIPs, podBIPs, err := createTestPods(data)
 	if err != nil {
 		t.Fatalf("Error when creating test Pods: %v", err)
 	}
 
+	svcB, err := createTestService(data, v6Enabled)
+	if err != nil {
+		t.Fatalf("Error when creating perftest-b Service: %v", err)
+	}
+	// Wait for the Service to be realized.
+	time.Sleep(3 * time.Second)
```

Comment:
Is there a simple way to check if service is realized? A fixed waiting interval either wastes time or stops too early.

Reply:
Hi Ziyou, this `time.Sleep` was introduced in https://github.com/antrea-io/antrea/pull/1815 first time, our tests inherit from `flowaggregator_test.go` keep it as well. I'm not sure if weiqiang found something while debugging so he added this wait statement. I could remove it here and `flowvisibility_test.go` to see if there will be any errors.